packages/components/src/components/tabs/tabs.tsx

```diff
@@ -1,39 +1,63 @@
 import classNames from 'classnames';
 import React from 'react';
-import { withRouter } from 'react-router-dom';
-import PropTypes from 'prop-types';
-import Tab from './tab.jsx';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import Tab from './tab';
 import { useConstructor } from '../../hooks';
 import ThemedScrollbars from '../themed-scrollbars/themed-scrollbars';
 
+type TTabsProps = RouteComponentProps & {
+    active_icon_color: string;
+    active_index?: number;
+    background_color: string;
+    bottom: boolean;
+    center: boolean;
+    children: React.ReactElement & React.ReactElement[];
+    className?: string;
+    fit_content: boolean;
+    has_active_line?: boolean;
+    has_bottom_line?: boolean;
+    header_fit_content: boolean;
+    history: History;
+    icon_color: string;
+    icon_size: number;
+    is_100vw: boolean;
+    is_full_width: boolean;
+    is_overflow_hidden: boolean;
+    is_scrollable: boolean;
+    onTabItemClick: (active_tab_index: number) => void;
+    should_update_hash: boolean;
+    single_tab_has_no_label: boolean;
+    top: boolean;
+};
+
 const Tabs = ({
     active_icon_color,
-    active_index,
+    active_index = 0,
     background_color,
     bottom,
     center,
     children,
-    className,
+    className = '',
```

Comment:
why do we need default value here? we can just skip it, and for Tab component mark also this className property like optional, as we do it everywhere in the code. How do you think? :)